src/extension.ts

```diff
@@ -735,7 +734,7 @@ function startDocu() {
     outputChannel.appendLine(`Path to the documentation: ${docuPath}`);
     outputChannel.appendLine(`Address to the website: ${docuAddress}`);
 
-    //open(docuAddress);
+    vscode.env.openExternal(vscode.Uri.parse(docuAddress));
```

Comment:
@jurekseverin-isg 

Here the entry in the changelog is missing

Reply:
It's already added at #Fixed within changelog, or do you mean something else?

Reply:
Sorry you are right